include/bundle.h

```diff
@@ -95,6 +96,9 @@ G_GNUC_WARN_UNUSED_RESULT;
  * @param bundle return location for a RaucBundle struct.
  *               This will contain all bundle information obtained by
  *               check_bundle
+ *               Will return NULL on error, except for R_BUNDLE_ERROR_CRYPT where
+ *               an incomplete bundle struct is returned to allow accessing
+ *               bundle->envelopeddata.
```

Comment:
We should discuss if this is needed.

Reply:
Came to the conclusion that we skip this part of the PR first of all.

As a result, the recipients can only be dumped if the decryption key is given.
A future adaptation might make this possible for missing decryption keys, too (which is probably quite useful for debugging decryption issues).